Schedule interviews according to candidate preferences (#1161)
sourcegraph/about: handbook/people-ops/interview_process.md

## Scheduling

> We are flexible when it comes to scheduling interviews because we are all-remote and we don't need to schedule all interviews back-to-back (unlike typical onsite interviews at other companies). Some candiates like spreading interviews out over multiple days and others prefer to batch them as much as possible to get it over with. *Ask the candidate what their preferences are and then try to accomidate those preferences as much as possible (given interviewer availability)*.

Reviewer: ```suggestion
We are flexible when it comes to scheduling interviews because we are all-remote and we don't need to schedule all interviews back-to-back (unlike typical onsite interviews at other companies). Some candiates like spreading interviews out over multiple days and others prefer to batch them as much as possible to get it over with. *Ask the candidate what their preferences are and then try to accommodate those preferences as much as possible (given interviewer availability)*.
```

> We are flexible when it comes to scheduling interviews because we are all-remote and we don't need to schedule all interviews back-to-back (unlike typical onsite interviews at other companies). Some candiates like spreading interviews out over multiple days and others prefer to batch them as much as possible to get it over with. *Ask the candidate what their preferences are and then try to accommodate those preferences as much as possible (given interviewer availability)*.

Reviewer 2: ```suggestion
We are flexible when it comes to scheduling interviews because we are all-remote and we don't need to schedule all interviews back-to-back (unlike typical onsite interviews at other companies). Some candiates like spreading interviews out over multiple days and others prefer to batch them as much as possible to get it over with. *Ask the candidate what their preferences are / if they prefer having a 30m break between interviews, and then try to accommodate those preferences as much as possible (given interviewer availability)*.
```

I think it's important to be explicit that is a preference they can express not just "all-on-one-day or spread-across-multiple-days

Author: Yeah not a binary choice. I tweaked your wording a bit.

```suggestion
We are flexible when it comes to scheduling interviews because we are all-remote and we don't need to schedule all interviews back-to-back (unlike typical onsite interviews at other companies). Some candidates like spreading interviews out (e.g., across multiple days and/or having breaks between interviews on the same day) and others prefer to batch them as much as possible to get it over with. *Ask the candidate what their ideal interview schedule looks like and then try to accommodate those preferences as much as possible (given interviewer availability)*.
```

Reviewer 2: great!
